beacon-chain/sync/validate_data_column.go

```diff
@@ -48,15 +46,19 @@ func (s *Service) validateDataColumn(ctx context.Context, pid peer.ID, msg *pubs
 	}
 
 	// Ignore messages that are not of the expected type.
-	ds, ok := m.(*eth.DataColumnSidecar)
+	dspb, ok := m.(*eth.DataColumnSidecar)
 	if !ok {
 		log.WithField("message", m).Error("Message is not of type *eth.DataColumnSidecar")
 		return pubsub.ValidationReject, errWrongMessage
 	}
+	ds, err := blocks.NewRODataColumn(dspb)
+	if err != nil {
+		return pubsub.ValidationReject, errors.Wrap(err, "roDataColumn conversion failure")
+	}
+	vf := s.newColumnVerifier(ds, verification.GossipBlobSidecarRequirements)
```

Comment:
Is it possible to have a more generic naming? HEre using `GossipBlobSidecareRequirements` feels hacky. (Same for requirements themselves.)

Reply:
We need it to be specific because blobs and data columns technically have different validation requirements(very minor differences)

Reply:
Yes the point here is we use `GossipBlobSidecareRequirements` for columns.

Reply:
Fixed